server/stream/messages/emitter.js

```diff
@@ -27,11 +28,28 @@ Meteor.startup(function() {
 		}
 	}
 
-	return Messages.on('change', function({ clientAction, id, data/* , oplog*/ }) {
+	// return Messages.on('change', function({ clientAction, id, data/* , oplog*/ }) {
+	// 	switch (clientAction) {
+	// 		case 'inserted':
+	// 		case 'updated':
+	// 			const message = data || Messages.findOne({ _id: id });
+	// 			publishMessage(clientAction, message);
+	// 			break;
+	// 	}
+	// });
+
+	return RoomEvents.on('change', function({ clientAction, id, data/* , oplog*/ }) {
 		switch (clientAction) {
 			case 'inserted':
 			case 'updated':
-				const message = data || Messages.findOne({ _id: id });
+				let message = data || RoomEvents.findOne({ _id: id });
```

Comment:
Can filter by `t` and prevent to return the data from the database if it's not a message type?